web/src/settings_org.js

```diff
@@ -692,6 +714,10 @@ export function discard_property_element_changes(elem, for_realm_default_setting
             set_realm_waiting_period_setting();
             break;
         default:
+            if (realm_group_dropdown_mapping.has(property_name)) {
+                set_dropdown_list_widget_setting_value(property_name, property_value);
+                break;
```

Comment:
Not sure if it is correct to include this in the `default` section since this is a special case where dropdown list widgets are used. We can probably update `set_input_element_value` and also `get_input_element_value` below, to handle the dropdown list widgets.

Reply:
Although, we may have many settings coverted to dropdown list widgets later and we may probably need to change it to something else then where most of the settings are dropdown list widget.